### PR TITLE
Changes directory for client files, server now runs and responds to POST

### PR DESCRIPTION
**Pull request by lancelafontaine**

### CODE REVIEW

File: src/server/generator.py [hunk 1]
```diff
@@ -37,10 +40,14 @@ def generate(program, language, colour):
...
+
     return final
 
 
-# if __name__ == "__main__":
-    
-#     character = generate("Computer Engineering", "Java", "Blue")
-#     print(character)
+if __name__ == "__main__":
+    character = generate("Computer Engineering", "Java", "Blue")
```
Comment (tamarindd) [inline src/server/generator.py:52]: why did you leave this here?

File: src/server/generator.py [hunk 2]
```diff
@@ -37,10 +40,14 @@ def generate(program, language, colour):
                skill_nums[1], language_skill,
                skill_nums[2], colour_skill)
 
+    print(final)
+    print(final)
+    print(final)
+    print(final)
```
Comment (tamarindd) [inline src/server/generator.py:46]: ???

**Review by tamarindd — APPROVED** [review]

File: src/server/generator.py (hunk 2 above)
Comment (lancelafontaine) [reply src/server/generator.py:46]: Did I really add this?? I don't remember doing that at all. 🤔
Will remove in next PR (I'm having final be a dictionary instead that is returned)

File: src/server/generator.py (hunk 1 above)
Comment (lancelafontaine) [reply src/server/generator.py:52]: No reason. Removed in next PR 👍 ,

**Pull request merged**